app/design/adminhtml/default/default/template/catalog/category/tree.phtml

```diff
@@ -30,8 +30,6 @@
     <?php echo $this->getStoreSwitcherHtml();?>
     <div class="tree-actions">
         <?php if($this->getRoot()): ?>
-            <?php //echo $this->getCollapseButtonHtml() ?>
-            <?php //echo $this->getExpandButtonHtml() ?>
```

Comment:
There are references in https://github.com/OpenMage/magento-lts/blob/76d6e6a0391a13269e3137bedbcf0c8b30472d9a/app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php#L117-L125

which are not used anywhere. Should that be deleted too?

Reply:
right, I checked and remove those 2 methods